gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,504 @@
+
+  The following is a sample for enabling DRA feature gates. 
+  Refer to the Kubernetes documentation for full details on `enabling DRA on your cluster <https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#enabling-dynamic-resource-allocation>`__.
+
+  .. literalinclude:: ./manifests/input/kubeadm-init-config.yaml
+    :language: yaml
+    :caption: Sample Kubeadm Init Config with DRA Feature Gates Enabled
+
+- The NVIDIA GPU Operator v25.3.0 or later installed with CDI enabled on all nodes and NVIDIA GPU Driver 565 or later.
+  
+  A sample Helm install command below includes enabling CDI with ``cdi.enabled=true``.
+  Refer to the install documentation for details on `enabling CDI <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#common-chart-customization-options>`__.
+
+  .. code-block:: console
+
+      $ helm install --wait --generate-name \
+              -n gpu-operator --create-namespace \
+              nvidia/gpu-operator \
+              --version=${version} \
+              --set cdi.enabled=true
+
+  Note if you want to install the GPU DRA Driver using pre-installed drivers, you must install NVIDIA GPU Driver 565 or later, the corresponding IMEX packages on GPU nodes, and disable the IMEX systemd service before installing the GPU Operator.
+  Refer to the documentation on `installing the GPU Operator with pre-installed drivers <https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html#pre-installed-nvidia-gpu-drivers>`__ for more details.
+
+
+Install the GPU DRA Driver with Helm
+====================================
+
+#. Add the NVIDIA Helm repository:
+
+   .. code-block:: console
+
+      $ helm repo add nvidia https://helm.ngc.nvidia.com/nvidia \
+          && helm repo update
+
+#. Install NVIDIA DRA Driver:
+
+   .. code-block:: console
+
+      $ helm install nvidia-dra-driver-gpu nvidia/nvidia-dra-driver-gpu \
+            --version="25.3.0-rc.1" \
+            --create-namespace \
+            --namespace nvidia-dra-driver-gpu \
+            --set nvidiaDriverRoot=/run/nvidia/driver \
+            --set nvidiaCtkPath=/usr/local/nvidia/toolkit/nvidia-ctk \
+            --set resources.gpus.enabled=false
+
+Common Chart Customization Options
+==================================
+
+The following options are available when using the Helm chart.
+These options can be used with ``--set`` when installing with Helm.
+
+The following table identifies the most frequently used options.
+To view all the options, run ``helm show values nvidia/nvidia-dra-driver-gpu``.
+
+.. list-table::
+   :widths: 20 50 30
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+     - Default
+
+   * - ``nvidiaDriverRoot``
+     - Specifies the driver root on the host.
+       For Operator managed drivers, use ``/run/nvidia/driver``.
+       For pre-installed drivers, use ``/``.

```

Comment:
Can we provide a _recommended_ mode of operation? 

Something like "Use Operator-managed drivers if in doubt".